### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sliu126**
High level GPT2 documentation added, word level prediction described, contact information added

# Overview

Update Readme for GPT2 Implementation, word level prediction descriptions and Shijia's contact information 

## Ticket

https://www.pivotaltracker.com/n/projects/2460065/stories/182171305

## Contributions

- High level GPT2 documentation added
- Word level prediction described
- Contact information added

## Test

- No code added in this PR. Should pass all existing unit tests.

### CODE REVIEW

File: bcipy/language/README.md [hunk 1]
```diff
@@ -16,4 +16,33 @@ The pretrained GPT2 language model is saved in [this folder on Google Drive](htt
...
+
+Furthermore, we use a unigram language model trained on the ALS Phraseset to interpolate with GPT2 for smoothing purpose, since the distributions given by GPT2 can be very "sharp", where most of the probability mass falls on the top few letters. Therefore, interpolating with a unigram model could give more probability to the lower ranked letters.
```
Comment (tab-cmd) [inline bcipy/language/README.md:32]: We should comment more on this `ALS Phraseset`. Where can it be found? Or if it's from our group, a little more background on its collection!

**Review by tab-cmd — APPROVED** [review]
Thank you for this documentation! I understand your model much better having read it

**Review by lawhead — APPROVED** [review]
Thanks, this is very helpful!

File: bcipy/language/README.md (hunk 1 above)
Comment (sliu126) [reply bcipy/language/README.md:32]: I got the ALS Phraseset from Betts. She should be able to provide more details :)

**Pull request merged**